src/main/java/com/minecolonies/network/messages/AssignmentModeMessage.java

```diff
@@ -28,7 +29,7 @@
      */
     public AssignmentModeMessage()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) It is a good practice to call super() in a constructor [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ACallSuperInConstructor)